app/models/team.rb

```diff
@@ -15,6 +15,14 @@ def slug_candidates
   has_many :invitations, class_name: "TeamInvitation"
   has_many :memberships, class_name: "TeamMembership"
   has_many :members, through: :memberships, source: :user
+  has_many :admins,
+    -> { where(team_memberships: { admin: true }) },
+    through: :memberships,
+    source: :user
 
   has_secure_token :token
+
+  def admin?(user)
```

Comment:
Could you rename this to `has_admin?` please. That parses better to me so in my head instead of saying  "Hello Team. Are you admin User:Karlo?" it instead says "Hello team, do you have User:Karlo as an admin?"